src/main/java/core/basesyntax/transactions/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.transactions;
+
+import core.basesyntax.storage.DateFruits;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void resultOfOperation(String fruitName, int amount) {
+        if (amount > 0) {
+            int currentAmount = DateFruits.get(fruitName);
+            int newAmount = currentAmount + amount;
+            DateFruits.save(fruitName, newAmount);
+
+        } else {
+            throw new RuntimeException(" The balance can`t be less or equals zero");
+        }
```

Comment:
better avoid if-else construction 
```suggestion
        if (amount <= 0) {
            throw new RuntimeException(" The balance can`t be less or equals zero");
        }
            int currentAmount = DateFruits.get(fruitName);
            int newAmount = currentAmount + amount;
            DateFruits.save(fruitName, newAmount);
```
fix it for other handlers